glib/src/value.rs

```diff
@@ -295,12 +299,7 @@ impl Drop for Value {
 
 impl fmt::Debug for Value {
     fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-        unsafe {
-            let s: GString =
-                from_glib_full(gobject_ffi::g_strdup_value_contents(self.to_glib_none().0));
-
-            f.debug_tuple("Value").field(&s).finish()
-        }
+        write!(f, "({}) {}", self.type_(), self.content_debug_string())
```

Comment:
If you feel like it, make `{:?}` print only the value and `{:#?}` with type (or the other way around). That's `Formatter::alternate()`

Reply:
I don't feel like it 😅

I would generally expect `#` to make things more verbose and here I wouldn't want to remove any verbosity from the regular debug output or add any for the alternate debug output.

Reply:
No strong opinion here, seems fine either way for me :)